<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherWindow</class>
 <widget class="QWidget" name="TeacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filterComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Фильтр по статусу</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditComment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshStatusPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить статус</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshTablePushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>600</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить таблицу</string>
   </property>
  </widget>
  <widget class="QComboBox" name="statusComboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
